.clang-format

```diff
@@ -40,7 +40,6 @@ IncludeCategories:
 IncludeIsMainRegex: '([-_](test|unittest))?$'
 IndentCaseLabels: true
 IndentWidth: 4
-InsertNewlineAtEOF: true
```

Comment:
Why not?

Reply:
Was having errors when I tried to run `clang-format` manually, when I removed this line it worked.

Reply:
What was the error?